charts/quickwit/values.yaml

```diff
@@ -199,6 +203,8 @@ metastore:
 
   serviceAnnotations: {}
 
+  serviceType: {}
```

Comment:
Same comment, we can have something like:

```yaml
metastore:
   service:
     # type: ClusterIP
     annotations: {}
```

Instead of `serviceAnnotations` and `serviceType`